index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+
+    <title>My portfolio</title>
+
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins:regular,bold,500&display=swap" />
+    <link rel="stylesheet" href="main.css" />
+</head>
+
+<body>
+    <div class="mobile-background">
+        <header class="app-bar">
+            <a href="/" class="logo"><img src="images/logo.svg" alt="Home" width="76" height="20" /></a>
+            <nav>
+                <a href="#menu" class="tertiary-button"><img src="images/icons/menu.svg" alt="Menu" width="24" height="24" /></a>
+            </nav>
+        </header>
+        <section class="headline">
+            <h1 class="header-2 color-n800 mb-12">I’m Elmar, Glad to see you!</h1>
+            <p class="body-2 color-n600 mb-12 drop-shadow">
+                I’m a software developer! I can help you build a product feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+            </p>
+            <a href="#connect" class="link-button body-2 bolder color-b300 mb-12 drop-shadow">
+        LET'S CONNECT
+      </a>
+            <ul class="social-media-icons">
+                <li>
+                    <a href="#"><img src="images/icons/twitter.svg" alt="Elmar's Twitter" class="social-icon" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/icons/linkedin.svg" alt="Elmar's LinkedIn" class="social-icon" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/icons/medium.svg" alt="mb" class="social-icon" /></a>
+                </li>
+                <li>
+                    <a href="#"><img src="images/icons/angel-list.svg" alt="xxx" class="social-icon" /></a>
+                </li>
+            </ul>
+        </section>
+    </div>
+    <section class="works grid-container">
```

Comment:
#I think it would be nicer to put your work in the main tag

Reply:
Dear @hiromataba,

Yes, you are right, I did not use the main tag in this project. I will review my HTML file and fix it.

My regards,

Elmar